app/globals.css

```
@@ -36,6 +29,62 @@ body {
   animation: fadeInUp 0.4s ease both;
 }
 
+@keyframes slideInUp {
```

Stylelint (17.12.0)

[error] Expected keyframe name "slideInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)